alacritty_terminal/src/tty/windows/child.rs

```diff
@@ -145,6 +154,10 @@ mod tests {
         poller.wait(&mut events, Some(WAIT_TIMEOUT)).unwrap();
         assert_eq!(events.iter().next().unwrap().key, PTY_CHILD_EVENT_TOKEN);
         // Verify that at least one `ChildEvent::Exited` was received.
-        assert_eq!(child_exit_watcher.event_rx().try_recv(), Ok(ChildEvent::Exited));
+        assert_eq!(
+            child_exit_watcher.event_rx().try_recv(),
+            Ok(ChildEvent::Exited(Some(1))),
+            "Killed process should report non-zero exit code"
```

Comment:
The message and assertion do not match. Should say we expect an exit code of `1` if that's what we're asserting.

I'd go so far and say this message provides no additional benefit and if we wanted to add a message the most useful thing to put there would be the exit status we actually got.